src/core/worker.js

```diff
@@ -555,12 +452,7 @@ var WorkerMessageHandler = {
             !fullRequest.isRangeSupported) {
           // If stream or range are disabled, it's our only way to report
           // loading progress.
-          fullRequest.onProgress = function (evt) {
-            handler.send('DocProgress', {
-              loaded: evt.loaded,
-              total: evt.total,
-            });
-          };
+          handler.send('DocProgress');
```

Comment:
How can this work if you're not actually sending any data with the message!?
I've quickly tested the viewer, and the progress bar is now broken, which I'm assuming might be connected to this!